packages/core/src/pill-input/PillInput.css

```diff
@@ -103,7 +103,7 @@
 }
 
 .saltPillInput-bordered.saltPillInput-focused .saltPillInput-activationIndicator {
-  border-bottom-width: calc(var(--salt-size-border-strong) - var(--salt-size-border));
+  border-bottom-width: calc(var(--salt-size-fixed-200) - var(--salt-size-fixed-100));
```

Comment:
As with other comments, I think I'm missing understanding this calc! It looks to me as if we're subtracting 1px from 2px when we could just make it `--salt-size-fixed-100`. I'm clearly not getting it! sorry if its obvious.